Makefile

```diff
@@ -257,6 +257,24 @@ clean-for-device-build:
 .PHONY: clean-test-with-device-farm
 clean-test-with-device-farm: clean-for-device-build test-with-device-farm
 
+
+.PHONY: test-with-firebase
+test-with-firebase:
+# Ignore the build part as we have build in fastlane, recomment out these part if test locally.
+# 	xcodebuild \
+# 		-workspace Apps/Apps.xcworkspace \
+# 		-scheme $(SCHEME) \
+# 		-enableCodeCoverage YES \
+# 		-configuration $(CONFIGURATION) \
+# 		-sdk iphoneos build-for-testing;
+	cd ${BUILD_DIR}/Build/Products; \
+	zip -r ${APP_NAME}.zip $(CONFIGURATION)-iphoneos $(SCHEME)_iphoneos*.xctestrun;
+	gcloud firebase test ios run --test ${BUILD_DIR}/Build/Products/${APP_NAME}.zip \
+		--timeout 20m \
+		--device model=iphone11,version=13.6 \
+		--results-dir $(SCHEME)-${CIRCLE_BUILD_NUM};
```

Comment:
Can we consider the usage of the Fastlane plugin – https://github.com/fastlane/fastlane-plugin-firebase_test_lab ? 
It would simplify firebase testing to a single fastlane call. Also, the full configuration would live in the single Fastfile.

Reply:
As discussed, will iterate this in the future.